runtime/src/main/java/org/corfudb/util/MetricsUtils.java

```diff
@@ -243,17 +242,14 @@ private static void setupJvmMetrics(@NonNull MetricRegistry metrics) {
     private static void setupLogAnalysisMetrics(@NonNull MetricRegistry metrics) {
         if (!metricsLogAnalysisEnabled) return;
 
-        final ILoggerFactory iLoggerFactory = LoggerFactory.getILoggerFactory();
-        try {
-            final Logger root = (Logger) iLoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
-            final InstrumentedAppender logMetrics = new InstrumentedAppender(metrics);
-            logMetrics.setName("logger.metrics");
-            logMetrics.setContext(root.getLoggerContext());
-            logMetrics.start();
-            root.addAppender(logMetrics);
-        } catch (ClassCastException e) {
-            log.warn("Logger is not logback compatible. Log analysis metrics are not collected.", e);
-        }
+        final Slf4jReporter reporter = Slf4jReporter.forRegistry(metrics)
+                .outputTo(LoggerFactory.getLogger("logger.metrics"))
+                .convertRatesTo(TimeUnit.SECONDS)
+                .convertDurationsTo(TimeUnit.MILLISECONDS)
+                .build();
+
+        //reporter.start(1, TimeUnit.MINUTES);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)